### PR TITLE
Update silversurfer.ms

### PR DESCRIPTION
**Pull request by Supertre03**

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 1]
```diff
@@ -9,6 +9,41 @@
...
-            'powers': array(),
-            'buffs': array()
+            'powers': array(
+                "/buff",
+                "/carryplayer",
+                "/dismountplayer",
+                "/jump 0-9",
+                "/run 0-9",
+                "/haste 0-9",
+                "/gmc",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: i need to see an example of him creating matter

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 2]
```diff
@@ -9,6 +9,41 @@
...
+                "/gms",
+                "/fly",
+                "/speak <playername>",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: mind controlling someone

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 3]
```diff
@@ -9,6 +9,41 @@
...
+                "/intangible",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: going intangible

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 4]
```diff
@@ -9,6 +9,41 @@
...
+                "/tangible",
+                "/invis",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: going invisible

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 5]
```diff
@@ -9,6 +9,41 @@
...
+                "/vis",
+                "/disguiseviewself",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: disguising as someone

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 6]
```diff
@@ -9,6 +9,41 @@
...
+                "/mindcontrol <playername>",
+                "/stopmindcontrol",
+                "/tp",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: teleporting

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 7]
```diff
@@ -9,6 +9,41 @@
...
+                "/gmc",
+                "/gms",
+                "/fly",
+                "/speak <playername>",
+                "/intangible",
+                "/tangible",
+                "/invis",
+                "/vis",
+                "/disguiseviewself",
+                "/mindcontrol <playername>",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: mind controlling someone

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 8]
```diff
@@ -9,6 +9,41 @@
...
+                "/stopmindcontrol",
+                "/tp",
+                "/heal",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: healing someone

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 9]
```diff
@@ -9,6 +9,41 @@
...
+                "/repair",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: matter manipulation

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 10]
```diff
@@ -9,6 +9,41 @@
...
+                "/skull",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: matter creation

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 11]
```diff
@@ -9,6 +9,41 @@
...
+                "/time",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: time control

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 12]
```diff
@@ -9,6 +9,41 @@
...
+                "/spawner",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: matter manipulation

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 13]
```diff
@@ -9,6 +9,41 @@
...
+                "/back",
+                "/top"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: teleportation

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 14]
```diff
@@ -9,6 +9,41 @@
...
+                "/disguiseviewself",
+                "/mindcontrol <playername>",
+                "/stopmindcontrol",
+                "/tp",
+                "/heal",
+                "/repair",
+                "/skull",
+                "/time",
+                "/spawner",
+                "/back",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: teleportation

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 15]
```diff
@@ -9,6 +9,41 @@
...
+                "/top"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 25 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: proof he is stronger than superman

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 16]
```diff
@@ -9,6 +9,41 @@
...
+                "/effect ". @plr ." 10 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: regeneration

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 17]
```diff
@@ -9,6 +9,41 @@
...
+                "/effect ". @plr ." 11 9999 3 true",
+                "/effect ". @plr ." 12 9999 1 true",
+                "/effect ". @plr ." 13 9999 1 true",
+                "/effect ". @plr ." 16 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: night vision

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 18]
```diff
@@ -9,6 +9,41 @@
...
+                "/effect ". @plr ." 21 9999 19 true",
+                "/give ". @plr ." minecraft:iron_boots 1 8:30 4:30 name:&4S&ep&4e&ce&ed&8s&0t&9e&1r_&4Boots"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: super running speed

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 19]
```diff
@@ -9,6 +9,41 @@
...
+                "/time",
+                "/spawner",
+                "/back",
+                "/top"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 25 true",
+                "/effect ". @plr ." 10 9999 1 true",
+                "/effect ". @plr ." 11 9999 3 true",
+                "/effect ". @plr ." 12 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: immunity to fire/lava

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 20]
```diff
@@ -9,6 +9,41 @@
...
             'op': 'true',
             'speech': "I am the Cosmic Surfer! Traveling through time since... um idk.",
-            'powers': array(),
-            'buffs': array()
+            'powers': array(
+                "/buff",
+                "/carryplayer",
+                "/dismountplayer",
+                "/jump 0-9",
+                "/run 0-9",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: super running speed

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 21]
```diff
@@ -9,6 +9,41 @@
             'trail': '',
             'op': 'true',
             'speech': "I am the Cosmic Surfer! Traveling through time since... um idk.",
-            'powers': array(),
-            'buffs': array()
+            'powers': array(
+                "/buff",
+                "/carryplayer",
+                "/dismountplayer",
+                "/jump 0-9",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Required Evidence: super jumping ability

**Comment by ZargonKing** [comment]
https://www.youtube.com/watch?v=RjGdDb7fAAQ

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms (hunk 2 above)
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Where is the evidence?

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms (hunk 3 above)
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Where is the evidence?

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms (hunk 5 above)
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Where is the evidence?

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms (hunk 6 above)
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Where is the evidence?

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 22]
```diff
@@ -9,6 +9,26 @@
...
+            'powers': array(
+                "/buff",
+                "/carryplayer",
+                "/dismountplayer",
+                "/jump 0-7",
+                "/run 0-5",
+                "/haste 0-9",
+                "/fly",
+                "/speak <playername>",
+                "/thru",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Evidence Required: Going through walls

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 23]
```diff
@@ -9,6 +9,26 @@
...
+                "/disguiseviewself",
+                "/tp",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Evidence Required: teleportation

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 24]
```diff
@@ -9,6 +9,26 @@
...
+                "/heal",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Evidence Required: healing

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 25]
```diff
@@ -9,6 +9,26 @@
...
+           ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 20 true",
+                "/effect ". @plr ." 10 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Evidence Required: regeneration

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 26]
```diff
@@ -9,6 +9,26 @@
...
+                "/effect ". @plr ." 11 9999 3 true",
+                "/effect ". @plr ." 12 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Evidence Required: immunity to lava and fire

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms (hunk 24 above)
Comment (Supertre03) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: I gave you evidence of this

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms (hunk 25 above)
Comment (Supertre03) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: If he can heal others he can heal himself! Well to me really it makes sense.

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms (hunk 22 above)
Comment (Supertre03) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: With his board he can go through walls.. and anything really.

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 27]
```diff
@@ -9,6 +9,24 @@
...
-            'buffs': array()
+            'powers': array(
+                "/buff",
+                "/carryplayer",
+                "/dismountplayer",
+                "/jump 0-7",
+                "/run 0-5",
+                "/haste 0-9",
+                "/fly",
+                "/speak <playername>",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: evidence of him mind controlling pls

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms (hunk 22 above)
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: show me the evidence pls

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 28]
```diff
@@ -9,6 +9,24 @@
...
+                "/thru",
+                "/disguiseviewself",
+                "/heal",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: EVIDENCE REQUIRED: healing

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms (hunk 27 above)
Comment (TreyCodes) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: Ight nigma damn

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 29]
```diff
@@ -9,6 +9,22 @@
...
+                "/buff",
+                "/carryplayer",
+                "/dismountplayer",
+                "/jump 0-7",
+                "/run 0-5",
+                "/haste 0-9",
+                "/fly",
+                "/thru",
+                "/disguiseviewself",
+                "/heal",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: EVIDENCE REQUIRED: healing

File: CommandHelper/LocalPackages/commands/buffs/silversurfer.ms [hunk 30]
```diff
@@ -9,6 +9,22 @@
...
-            'buffs': array()
+            'powers': array(
+                "/buff",
+                "/carryplayer",
+                "/dismountplayer",
+                "/jump 0-7",
+                "/run 0-5",
+                "/haste 0-9",
+                "/fly",
+                "/thru",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/silversurfer.ms]: EVIDENCE REQUIRED: intangibility

**Comment by Supertre03** [comment]
https://www.youtube.com/watch?v=RjGdDb7fAAQ

**Comment by Supertre03** [comment]
https://www.youtube.com/watch?v=xZfULu7pAqY is Intangibility evidence, its right at the start

**Pull request merged**